tests/unit/test_main.py

```diff
@@ -1807,3 +1807,57 @@ async def test_delete_tag_range_type_error(self, event_loop):
         url = f"{self.base_endpoint}/interfaces/{interface_id}/tag_ranges"
         response = await self.api_client.delete(url)
         assert response.status_code == 400
+
+    async def test_get_all_tag_ranges(self, event_loop):
+        """Test get_all_tag_ranges"""
+        self.napp.controller.loop = event_loop
+        dpid = '00:00:00:00:00:00:00:01'
+        switch = get_switch_mock(dpid)
+        interface = get_interface_mock('s1-eth1', 1, switch)
+        tags = {'vlan': [[1, 4095]]}
+        interface.tag_ranges = tags
+        interface.available_tags = tags
+        switch.interfaces = {1: interface}
+        self.napp.controller.switches = {dpid: switch}
+        url = f"{self.base_endpoint}/interfaces/tag_ranges"
+        response = await self.api_client.get(url)
+        expected = {dpid: {
+            '1': {
```

Comment:
@Alopalao your last commits look good, but this unit test will fail, it needs to be refactored too. It probably didn't catch in the CI yet since it's on top of other PR. Waiting for this fix before approving.

Reply:
Forgot to add coverage back to tox, fixed